crates/astria-composer/src/searcher/collector.rs

```diff
@@ -32,6 +32,10 @@ pub(super) struct Transaction {
     pub(super) inner: ethers::types::Transaction,
 }
 
+/// A Collector is a sub-actor in the Searcher module that interfaces with individual rollups.
+/// It is responsible for fetching pending transactions submitted to the rollup nodes and then
+/// passing them downstream for the searcher to process. Thus, a searcher can have multiple
+/// collectors running at the same time funneling data from multiple rollup nodes.
```

Comment:
Cleaned up language a bit here, but main things:
- collector doesn't fetch right now, future iterations could but collect is the most accurate.
- each collector operates on a specific rollup node. 

```suggestion
/// Collector is a sub-actor in the Searcher module that interfaces with
/// individual rollups. It collects transactions submitted to a rollup node and
/// passes them downstream for the searcher to process.
/// 
/// A searcher may have multiple collectors running at the same time funneling
/// data from multiple rollup nodes.
```